tests/integration/test_wpk/test_wpk_manager.py

```diff
@@ -925,10 +934,10 @@ def test_wpk_manager(set_debug_mode, get_configuration, configure_environment,
         remove_wpk_package()
 
     # Give time for registration key to be available and send a few heartbeats
-    time.sleep(40)
+    time.sleep(time_until_registration_key_avaible)
 
     # Send upgrade request
-    response = send_message(data, UPGRADE_SOCKET)
+    response = json.loads(upgrade_socket.send(json.dumps(data)))
```

Comment:
Could we hide the dumps and loads inside the `WazuhSocket` class? Here are some options I think could work:
- Use a couple of internal configuration parameters set when the object is instantiated
- Check if the type of the data passed in is an instance of a dictionary / try to load the reponse.
- Use a couple optional parameters to the send method to control handing of data.

Reply:
Done in 067df572cd4272d078dc144bf46d2c0bd1d9feee and da33011e6e3b1f1be28d531f3982cb5142bb4bed